source/_components/sensor.magicseaweed.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Magicseaweed Sensor"
+description: "How to integrate Magicseaweed within Home Assistant."
+date: 2018-06-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: magicseaweed.png
```

Comment:
This logo is missing in the repository, maybe you forgot to add it?

Reply:
Yes, I did forget to add it. I'm still new at this. What's the procedure for adding it / where does it go?

Reply:
Just added. A transparent background image was not available. According to their API documentation, they request that the logo I uploaded be used.